tests/desktop/consumer_pages/test_reviews.py

```diff
@@ -41,10 +41,6 @@ def _create_review(self, mozwebqa):
         details_page = add_review_box.write_a_review(mock_review['rating'], mock_review['body'])
 
         # Step 4 - Check review
-        details_page.wait_notification_box_visible()
```

Comment:
I moved them into the page object. There's no need to wait for the notification to be hidden though, that just adds to the test run time unnecessarily.